hazelcast/src/main/java/com/hazelcast/concurrent/atomicreference/operations/AlterAndGetOperation.java

```diff
@@ -37,12 +37,16 @@ public void run() throws Exception {
         IFunction f = nodeEngine.toObject(function);
         AtomicReferenceContainer atomicReferenceContainer = getReferenceContainer();
 
-        Object input = nodeEngine.toObject(atomicReferenceContainer.get());
+        Data originalData = atomicReferenceContainer.get();
+        Object input = nodeEngine.toObject(originalData);
         //noinspection unchecked
         Object output = f.apply(input);
-        shouldBackup = true;
-        backup = nodeEngine.toData(output);
-        atomicReferenceContainer.set(backup);
+        Data serializedOutput = nodeEngine.toData(output);
```

Comment:
I would call serializedOutput somehting like outputData. So it has 'Data' in the name